### PR TITLE
Fix incorrect tests : totalLen field of ipv4 header, action name in add entry call

### PR DESCRIPTION
**Pull request by usha1830**

### CODE REVIEW

File: testdata/p4_16_samples/pna-dpdk-parser-state-err.p4 [hunk 1]
```diff
@@ -172,7 +172,7 @@ control MainControlImpl(
         hdr.ipv4.srcAddr = newAddr;
     }
     action add_on_miss_action2() {
-        add_entry(action_name="next_hop", action_params = {32w0, 32w1234}, expire_time_profile_id = user_meta.timeout);
+        add_entry(action_name="next_hop2", action_params = {32w0, 32w1234}, expire_time_profile_id = user_meta.timeout);
```
Comment (psivanup) [inline testdata/p4_16_samples/pna-dpdk-parser-state-err.p4:175]: Is this action_name change required for the issue 3636?

**Review by psivanup — COMMENTED** [review]
Please check.

File: testdata/p4_16_samples/pna-dpdk-parser-state-err.p4 (hunk 1 above)
Comment (usha1830) [reply testdata/p4_16_samples/pna-dpdk-parser-state-err.p4:175]: No. It is coming from another for add_entry (PR 3614)

**Review by psivanup — APPROVED** [review]
Looks good to me.

**Comment by usha1830** [comment]
Closing as upstreamed.

**Pull request closed without merging**